catalogue-service-micro/src/main/java/eu/nimble/service/catalogue/impl/CatalogueLineController.java

```diff
@@ -97,6 +98,9 @@ public ResponseEntity getCatalogueLineByHjid(@ApiParam(value = "Identifier of th
             produces = {"application/json"},
             method = RequestMethod.GET)
     public ResponseEntity getCatalogueLinesByHjids(@ApiParam(value = "Identifiers of the catalogue lines to be retrieved. (line.hjid)", required = true) @RequestParam(value = "ids") List<Long> hjids,
+                                                   @ApiParam(value = "Number of catalogue lines to be included in CataloguePaginationResponse ") @RequestParam(value = "limit",required = true) Integer limit,
+                                                   @ApiParam(value = "Offset of the first catalogue line among all catalogue lines of the default catalogue for the party") @RequestParam(value = "offset",required = true) Integer offset,
```

Comment:
This line also should be updated:@ApiParam(value = "Offset of the first catalogue line among all found catalogue lines") @RequestParam(value = "offset",required = true) Integer offset

Reply:
Done